tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -54,15 +54,28 @@
         width:30%;
       }
       .right-pane {
-        width:70%;
+        width:100%;
         height:300px;
       }
       .table-style {
         table-layout:fixed;
         width:95%;
       }
       .performance-table {
-        width: 100%;
+        width:100%;
+      }
+      .x-axis-title {
+        text-align:center;
+      }
+      .y-axis-title {
+        writing-mode:bt-rl;
+        -webkit-transform:rotate(270deg);
+        -moz-transform:rotate(270deg);
+        -o-transform: rotate(270deg);
+        -ms-transform:rotate(270deg);
+        transform: rotate(270deg);
```

Comment:
Could we just use `transform` and remove the other platform-specific properties? Firefox and Chrome both support `transform`.

Reply:
done.